TACC/Core-Portal
# task/TV3-155, task/TV3-156: Customizable Tapis v2 Jobs View

## Overview

- Add `jobsv2Title` param to `_WORKBENCH_SETTINGS` to determine label for historic tapis v2 jobs view
- Conditionally render the `https://cep.test/workbench/history/jobsv2` path based on bool of `jobsv2Title`
- Change render of no job results to `<SectionMessage>`

## Related

* [TV3-155](https://jira.tacc.utexas.edu/browse/TV3-156)


## Testing

1. Alter `jobsv2Title` in `_WORKBENCH_SETTINGS` to a different string
2. Go to https://cep.test/workbench/history/jobsv2 and confirm label matches your value
3. Remove `jobsv2Title` and confirm at https://cep.test/workbench/history/ there is no jobsv2 item in the sidebar
4. Confirm response for no results appears as shown below

## UI

![Screen Shot 2023-04-11 at 12 45 52 PM](https://user-images.githubusercontent.com/20326896/231246960-df7964f1-0fb6-492a-9de1-5c9cffa3991e.png)
![Screen Shot 2023-04-11 at 12 45 45 PM](https://user-images.githubusercontent.com/20326896/231246963-8cbe4e9f-e697-46a6-b762-282699bbb4e9.png)
![Screen Shot 2023-04-11 at 12 33 26 PM](https://user-images.githubusercontent.com/20326896/231246964-c631e2f8-805d-4f36-bd3a-fb5621743a79.png)
